.babelrc

```diff
@@ -5,12 +5,10 @@
       "@babel/env",
       {
         "targets": {
-          "node": "14.17.3"
+          "node": "20.14.0"
         }
       }
     ]
   ],
-  "plugins":[
-    "@babel/plugin-proposal-object-rest-spread"
-  ]
-}
\ No newline at end of file
+  "plugins": ["@babel/plugin-proposal-object-rest-spread"]
```

Comment:
Don't think we need this:
![Screenshot 2024-06-17 at 09 56 15](https://github.com/guardian/fulfilment-lambdas/assets/181371/026331d1-40f3-4195-96a0-f6e68d5c0079)

Reply:
Removed thanks.